kaizen/auto/__init__.py

```
@@ -0,0 +1,243 @@
+    # Initialize Phoenix tracer
+    try:
+        from phoenix.otel import register
+
+        tracer_provider = register(
+            project_name=project,
+            endpoint=endpoint,
+        )
+        _tracer_provider = tracer_provider
+        logger.info(f"Phoenix tracing enabled: project={project}, endpoint={endpoint}")
+    except ImportError as e:
+        logger.exception(f"Phoenix not available: {e}. Install with: pip install arize-phoenix")
```

Ruff (0.14.14)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)